manifest.json

```diff
@@ -1,46 +0,0 @@
-{
-  "manifest_version": 2,
-
-  "name": "codesy.io",
-  "description": "codesy is a pay-what-you-want market for the open source community to encourage coders to fix important bugs.",
-  "version": "1.0",
-
-  "icons": {
-    "16": "img/icon16.png",
-    "48": "img/icon48.png",
-   "128": "img/icon128.png"
-  },
-
-  "permissions": [
-    "https://api.codesy.io/",
-    "storage",
-    "tabs",
-    "webNavigation"
-  ],
-  "options_page":"options.html",
-
-  "web_accessible_resources": [
-      "js/jquery-2.0.3.min.js",
-      "js/jquery-2.0.3.min.map"
-  ],
-  "background": {
-    "scripts": ["js/watch_xhr.js"],
-    "persistent": false
-  
-  },
-  "content_scripts" : [
-    {
-      "matches" : ["*://*.github.com/*"],
-      "css" : [
-        "css/codesy.css"
-      ],
-      "js" : [
-        "js/jquery-2.0.3.min.js",
-        "js/codesy.js"
-      ],
-      "all_frames" : true
-    }
-  ]
-
-}
-
```

Comment:
This removes manifest.json but I don't see the `prod/manifest.json` that seems like it should replace it?